_data/simple-icons.json

```
@@ -5348,6 +5348,11 @@
             "hex": "FABF15",
             "source": "https://www.instructables.com/community/Official-Instructables-Logos-1/"
         },
+        {
+            "title": "Instructure",
+            "hex": "2A7BA0",
+            "source": "https://www.instructure.com/"
```

```suggestion
            "source": "https://www.instructure.com/about/brand-guide/download-logos",
            "guidelines": "https://www.instructure.com/canvas/resources/noram-guides/instructure-brand-guide-2022"
```

Their logo resource and brand usage guideline can be found here.

---

I have made the necessary changes